test_supabase.py

```
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+"""
+Test script to verify Supabase integration for AI Ingesting Tool.
+"""
+
+import os
+import sys
+sys.path.append(os.path.dirname(os.path.abspath(__file__)))
+
+from video_ingest_tool.supabase_config import verify_connection, get_database_status
+from video_ingest_tool.auth import AuthManager
```

Ruff (0.11.9)

`video_ingest_tool.auth.AuthManager` imported but unused

Remove unused import: `video_ingest_tool.auth.AuthManager`

(F401)